src/PowerShellEditorServices/Services/DebugAdapter/Handlers/ScopesHandler.cs

```diff
@@ -20,14 +20,17 @@ internal class ScopesHandler : IScopesHandler
 
         public Task<ScopesResponse> Handle(ScopesArguments request, CancellationToken cancellationToken)
         {
-            VariableScope[] variableScopes =
-                _debugService.GetVariableScopes(
-                    (int)request.FrameId);
+            //We have an artificial breakpoint label, so just copy the stacktrace from the first stack entry for this.
```

Comment:
```suggestion
            // We have an artificial breakpoint label, so just copy the stacktrace from the first stack entry for this.
```

Could ou explain why this is either 0, or (and this is what's changed) `FrameId - 1`? That means there are two instances where we get scope 0: `FrameId == 0` _and_ `FrameId == 1`.

Reply:
Uh, there was a reason, but I honestly don't remember at this point. Let me check

Reply:
OK, I remember now, it's done so that the variables populate correctly in the debug pane for the label. I've added more of a comment explaining this.